docs/ui/insight/examples.md

```diff
@@ -1,17 +1,54 @@
 ---
-title: "Manage streams and consumer groups in RedisInsight"
-linkTitle: Streams
-description: Learn how to manage streams and consumer groups in RedisInsight
-weight: 1
+title: "RedisInsight examples"
+linkTitle: Examples
+description: 
+weight: 60
 ---
 
+
+RedisInsight allows you to browse, filter, and visualize key-value Redis data structures. It supports CRUD operation for lists, hashes, strings, sets, sorted sets, and so on. You can take advantage of built-in examples in Workbench to learn about the data structures. We'll guide you through one of those examples here and show you other practical ways to use RedisInsight with your databases.
+
+## Use RedisInsight to manage time series 
+
+Examples you find in Workbench allow you interact with some sample data in a few taps. For this orientation, we will use an example called _Redis for time series_. If you haven't done so, install [Redis Stack](/docs/stack/get-started/install). Then navigate to this example in Workbench, read the introduction, then tap the first button to create five time series for bike shops. 
+
+**Step 1: Create a time series per bike shop**
+
+```bash
+ TS.CREATE bike_sales_1 DUPLICATE_POLICY SUM LABELS region east compacted no
```

Comment:
While not a big fan of duplicating this (as you know) I believe it should focus on how to do this in RedisInsight.  It would be great to show a screenshot here where to paste this into RedisInsight e.g. Or where to find this guide